packages/docusaurus-plugin-content-blog/src/feed.ts

```diff
@@ -106,6 +106,14 @@ async function defaultCreateFeedItems({
       const $ = cheerioLoad(content);
 
       const link = normalizeUrl([siteUrl, permalink]);
+
+      $(`div#${blogPostContainerID} a`).each((_, elm) => {
+        const {href} = elm.attribs;
+        if (href) {
+          elm.attribs.href = String(new URL(href, link));
+        }
+      });
```

Comment:
LGTM 👍 

We will also need to convert image links to absolute, see 
https://github.com/facebook/docusaurus/issues/9136#issuecomment-1645593662

https://validator.w3.org/feed/docs/warning/ContainsRelRef.html

Reply:
Done. Image links now are absolutized.